src/serial_list_ports.cpp

```diff
@@ -123,21 +123,20 @@ bool handleEntry(const fs::directory_entry& entry,
 
 } // namespace
 
-extern "C" int serialGetPortsInfo(void (*callback_fn)(const char* port,
-                                                      const char* path,
-                                                      const char* manufacturer,
-                                                      const char* serialNumber,
-                                                      const char* pnpId,
-                                                      const char* locationId,
-                                                      const char* productId,
-                                                      const char* vendorId),
-                                  ErrorCallbackT error_callback)
+extern "C" int serialListPorts(void (*callback_fn)(const char* port,
+                                                   const char* path,
+                                                   const char* manufacturer,
+                                                   const char* serialNumber,
+                                                   const char* pnpId,
+                                                   const char* locationId,
+                                                   const char* productId,
+                                                   const char* vendorId),
+                               ErrorCallbackT error_callback)
```

Comment:
Why is this formatting still so f*ked up?

```suggestion
extern "C" int serialListPorts(
    void (*callback_fn)(
        const char* port,
        const char* path,
        const char* manufacturer,
        const char* serialNumber,
        const char* pnpId,
        const char* locationId,
        const char* productId,
        const char* vendorId
    ),
    ErrorCallbackT error_callback
)
```

Reply:
this is technically just one long type

Reply:
will replace it with a using